### ISSUE TITLE
Wrong command to create project

### ISSUE THREAD

**Issue by AliRezaBeigy** [react-native-community/react-native-template-typescript#190]
I'm going to create react native project with Typescript template using the following command:
```shell
npx react-native init MyApp --template react-native-template-typescript
```

As documented in [react native dev site](https://reactnative.dev/docs/typescript) and [this repository readme](https://github.com/react-native-community/react-native-template-typescript/blob/master/README.md)

but the command failed with the following error:
> error An unexpected error occurred: "https://registry.yarnpkg.com/react-native-template-react-native-template-typescript: Not found"

I try the following command everything work but still the project aren't typescript (there is a template folder contain typescript file)
```shell
npx react-native init MyApp --template typescript
```

**Comment by radko93** [comment]
You are unfortunately using a wrong CLI version. Please uninstall it and try to use the new one.
More details: https://github.com/react-native-community/react-native-template-typescript#note-on-the-legacy-cli and https://github.com/react-native-community/react-native-template-typescript/issues/80#issuecomment-536419979
